.tools/readmes/includes/prerequisites.jinja2

```diff
@@ -20,9 +20,12 @@ sample Hello World-style application.
 
 Next, for information on code example structures and how to build and run the examples, see [Getting started with the AWS SDK for C++ code examples](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started-code-examples.html).
 
+{% endif %}
+{% if service == 'bedrock' %}
```

Comment:
This feels like it should be part of the service metadata, so we can extend it for other services, instead of a custom block here in the template.

Reply:
Maybe? I don't really want to do that much refactoring today, though, and we don't really have anything in place to support that change.